communities/ucf-new-to-research/community-assessment.md

```diff
@@ -0,0 +1,45 @@
+# Community Assessment: Undergraduate students conducting Computer Science research at UCF
+
+## Definition
+* Undergraduate students either interested in conducting or already conducting Computer Science research at the University of Central Florida.
+
+## Size
+* 140 members of Mad Scientists Research Society (MSRS) Facebook group (the only student-led organization for undergraduate research at UCF); since the group is not specific to Computer Science, the actual number is likely much smaller.
+* No information on number of students overall that are involved/interested in CS research.
+* Would like to have between 200-300 people in the community given the size of the school, or, as many as possible.
+
+## Resources
+* MSRS has limited funding. The group runs a mentorship program for junior/seniors to assist freshman and sophomores. The club does hold meetings, but infrequently. The club also runs a day of student research talks each semester. More experienced students in the organization are resources for less experienced students.
+* Our ACM chapter has a SigAI group that holds workshops on AI-specific topics applicable to research, but no workshops on how to secure funding, network with professors, etc.
+* Aside from the club, UCF has a multitude of funding opportunities and programs available to undergraduate students. UCF also holds several workshops every semester on how to get involved in undergraduate research.
+* UCF Honors College holds many small workshops specific to publishing and networking.
+* Research talks are frequently given by faculty, visiting faculty, and graduate students.
+
+## Accessibility
+* MSRS has supportive officers willing to help any members, but the club as a whole has low visibility in the UCF community compared to other clubs. While the officers are helpful, executive board meetings are not accessible to the rest of the club and there is no clear path to officership. There are only a few club meetings a semester, which makes club involvement difficult if a member is working or cannot attend one of the meetings. No resources exist in the club that cater to older or less traditional students; i.e., no childcare is available and discussions tend to be focused on the traditional 4-year degree.
```

Comment:
I added some extra content in the Accessibility section to explain this. To summarize it, I'd say our club's low visibility is due to the low-prioritization of advertising since there isn't a dedicated leadership role for promoting the club. I think the closed executive meetings come from the club's perspective on what the role of officers is- it's thought of more as providing a service to members and less as leaders in a community where all members contribute to club management.